content/backend/graphql-js/4-adding-a-database.md

```diff
@@ -7,189 +7,235 @@ answers: ["To increase performance of the server", "When using two APIs, the Gra
 
-Here's an overview of the architecture that's used when building GraphQL servers with Prisma:
+Speaking of being productive and building awesome stuff, let's jump in and continue with our HackerNews Clone! 🏎💨
 
-![](https://imgur.com/OyIQQxF.png)
+### Setting up our project with Prisma and SQLite 
 
-The Prisma server provides the _data access layer_ in your application architecture, making it easy for your API server to talk to the database through Prisma. The API of the Prisma server is consumed by the Prisma client inside your _API server_ implementation (similar to an ORM). The API server is what you've started building throughout the previous chapters using `graphql-yoga`.
+<Instruction>
 
-In essence, Prisma lets you easily connect the GraphQL resolvers in your API server with your database.
+First, let's install the Prisma CLI by running the following command in your terminal:
 
-### Setting up Prisma with a demo database
+```bash(path=".../hackernews-node/")
+npm install @prisma/cli --save-dev
+```
 
-In this tutorial, you're going to build everything entirely from scratch! For your Prisma configuration, you're going to start with the most minimal setup that's possible.
+</Instruction>
 
-The first thing you need to do is create two files, which you're going to put into a new directory called `prisma`.
+Then, update your project structure.
 
 <Instruction>
 
-First, create the `prisma` directory and then two files called `prisma.yml` and `datamodel.prisma` by running the following commands in your terminal:
+Create the `prisma` directory and then a file called `schema.prisma` by running the following commands in your terminal:
 
 ```bash(path=".../hackernews-node/")
 mkdir prisma
-touch prisma/prisma.yml
-touch prisma/datamodel.prisma
+touch prisma/schema.prisma
 ```
 
 </Instruction>
 
-[`prisma.yml`](https://www.prisma.io/docs/-5cy7/) is the main configuration file for your Prisma setup. `datamodel.prisma` on the other hand contains the definition of your [datamodel](https://www.prisma.io/docs/-knul). The Prisma datamodel defines your application's _models_. Each model will be mapped to a table in the underlying database.
+Remember the GraphQL schema that you've been working with until now? Well, Prisma has a schema, too! You can think of the `prisma.schema` file as a *database schema*. It has three components:
+
+1. **Data source**: Specifies your database connection.
+1. **Generator**: Indicates that you want to genenerate Prisma Client.
+1. **Data model**: Defines your application *models*. Each model will be mapped to a table in the underlying database.
+
+Prisma's unique data model bridges the gap to help you reason about your data in a way that maps very well to the underlying database, while still providing an abstraction that allows you to be productive with type safety and superb auto-completion.
```

Comment:
While I agree with the sentiment, I'd remove the "superb", feels a little too marketing-y here 🙈

Reply:
hahah I felt conflicted about it as I was writing it 🙈 it really is bloody superb, but I understand 🙂